en/docs/apis/restapis/idp.yaml

```diff
@@ -2132,7 +2132,7 @@ components:
       properties:
         authenticatorId:
           type: string
-          example: U0FNTDJBdXRoZW50aWNhdG9y
+          example: U0FNTFNTT0F1dGhlbnRpY2F0b3I
```

Comment:
The base64 encoding of `SAMLSSOAuthenticator` gives `U0FNTFNTT0F1dGhlbnRpY2F0b3I=`. `=` sign is missing is it intentional?

![image](https://user-images.githubusercontent.com/84217321/203519402-85c7714d-f978-4921-9ce5-98ebad9950da.png)

Reply:
The equals sign "=" represents a padding, usually seen at the end of a Base64 encoded sequence.
Therefore, these two values are the same.